### ISSUE TITLE
bindfs file descriptor leak with --block-devices-as-files

### ISSUE THREAD

**Issue by mafanasyev-tri** [mpartel/bindfs#125]
It looks like bindfs leaks file descriptors when `--block-devices-as-files` used. Reproducer:
```
sudo mkdir -p /mnt/a
sudo bindfs /dev /mnt/a --block-devices-as-files
seq 2000 | xargs -i blkid /mnt/a/sda >/dev/null
ls -la /mnt/a
```
Last command fails with: `ls: reading directory '/mnt/a': Too many open files`
`strace` shows: 
```
openat(AT_FDCWD, "/mnt/a", O_RDONLY|O_NONBLOCK|O_CLOEXEC|O_DIRECTORY) = 3
fstat(3, {st_mode=S_IFDIR|0755, st_size=5000, ...}) = 0
getdents64(3, 0x5584a22e80d0, 32768)    = -1 EMFILE (Too many open files)
```
turns out `bindfs` process really did run out of file descriptors:
```
$ sudo lsof -c bindfs | wc -l
1033
$ sudo lsof -c bindfs | tail
bindfs  1430992 root 1015r   BLK    8,0      0t0      358 /dev/sda
bindfs  1430992 root 1016r   BLK    8,0      0t0      358 /dev/sda
bindfs  1430992 root 1017r   BLK    8,0      0t0      358 /dev/sda
bindfs  1430992 root 1018r   BLK    8,0      0t0      358 /dev/sda
bindfs  1430992 root 1019r   BLK    8,0      0t0      358 /dev/sda
bindfs  1430992 root 1020r   BLK    8,0      0t0      358 /dev/sda
bindfs  1430992 root 1021r   BLK    8,0      0t0      358 /dev/sda
bindfs  1430992 root 1022r   BLK    8,0      0t0      358 /dev/sda
bindfs  1430992 root 1023r   BLK    8,0      0t0      358 /dev/sda
```

This was discovered on Ubuntu 20.04, so bindfs 1.14.1 and fuse 2.9.9-3,
but I've checked the latest master (6d46e55a59b / 1.17.1) and problem is fully reproducable there as well.

**Comment by mpartel** [comment]
Thanks! Should be fixed in master now. I'll do a release a bit later.

**Comment by mpartel** [comment]
Fix released as 1.17.2
